module.lua

```diff
@@ -7,19 +7,15 @@ local _TEAM = {
 	fs = {
 		["Alfiecakes#0000"] = "EN",
 		["Arki#0113"] = "BR",
+		["Ashbolt#0000"] = 
```

Comment:
It's missing their community, lol

```suggestion
		["Ashbolt#0000"] = "EN",
```